impl/src/main/java/org/glassfish/soteria/servlet/SamRegistrationInstaller.java

```diff
@@ -59,7 +59,6 @@
 import org.glassfish.soteria.cdi.CdiExtension;
 import org.glassfish.soteria.cdi.spi.CDIPerRequestInitializer;
 import org.glassfish.soteria.cdi.spi.impl.LibertyCDIPerRequestInitializer;
-import org.glassfish.soteria.mechanisms.jaspic.ContextAuthenticationMechanismMapping;
 import org.glassfish.soteria.mechanisms.jaspic.HttpBridgeServerAuthModule;
 import org.glassfish.soteria.mechanisms.jaspic.Jaspic;
 
```

Comment:
Payara copyright may need to be added

Reply:
Will add it in another PR, because while GH allowed me to create new branch instead of forking repo, it will not allow me to push another commit onto this branch :)